release/apparmor_test.go

```diff
@@ -73,3 +73,14 @@ func (s *apparmorSuite) TestProbeAppArmorFullAppArmor(c *C) {
 	c.Check(level, Equals, release.FullAppArmor)
 	c.Check(summary, Equals, "apparmor is enabled and all features are available")
 }
+
+func (s *apparmorSuite) TestInterfaceSystemKey(c *C) {
+	fakeSysPath := c.MkDir()
+	restore := release.MockAppArmorFeaturesSysPath(fakeSysPath)
+	defer restore()
+	err := os.MkdirAll(filepath.Join(fakeSysPath, "policy"), 0755)
```

Comment:
Can you add "network" alongside policy please.

Reply:
Sure, done.